packages/sources/svelte/src/components/indicators/VtmnPrice/VtmnPrice.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+  import { VARIANT } from './enums';
+
+  /**
+   * ID of the price
+   * @type {string}
+   * @defaultValue undefined
+   */
+  export let id;
+
+  /**
+   * Label apply on the component
+   * @type {string}
+   * @defaultValue undefined
+   */
+  export let label;
```

Comment:
As the label will be inside the `<slot />`, is it possible to delete and set your `aria-label` manually when you use the components by spreading `$$restProps` as you we did in other Svelte components?
Thanks!